src/renderer/utils/storageHelper.ts

```diff
@@ -101,7 +101,7 @@ export class StorageHelper<T> {
 
   @action
   private configureObservable(options = this.options.observable) {
-    this.data = observable.box<T>(this.data.get(), {
+    this.data = observable.box<T>(this.options.defaultValue, {
```

Comment:
This looks really wrong imho.
Default value should be set only in `constructor()` and in `reset()`.

Reply:
I did this because you didn't want me changing `StorageHelper` too much. I would have gotten rid of this function completely otherwise.

Reply:
```
constructor:
    this.configureObservable(); // create observable field
    this.reset(); // default value is set here
```

What is wrong here? Why default value not set properly you think?

Reply:
Otherwise I would better to update it like `configureObservable({value?, options?})` and pass `defaultValue` in constructor.

Reply:
I remember being confused the first time I reviewed this function. It is (only) called by the constructor. So it is setting the initial value of `this.data`. It passes `this.data.get()` to the `observable.box<T>` constructor?? Maybe you intend that `configureObservable()` be called outside the constructor as well sometime(?) but seems weird for the constructor to use it.
Why not call `this.reset()` before `this.configureObservable()` in the constructor?

Reply:
We can remove (move) this function to constructor. It's organized in separated method for readability initially.
And yes, calling `this.reset()` first also make sense if it will help.

Reply:
If we call reset first, why are we "initializing" it twice anyway?

Reply:
Yeah I know that it will just be overwritten, but from a maintenance perspective we shouldn't leave these sorts of things around because the next reader will have to figure out if we actually want to use that inline declared value (we don't).

Reply:
I'm sure I am :)

Reply:
Ok, let's just get rid of `configureObservable` and put contents inside constructor.
That way init `this.options.defaultValue` make sense (if it's not locked in the method).